### PR TITLE
avoid send duplicated metrics data (close #215)

### LINKED ISSUE #215: `EaseMonitorMetrics` send the same metrics data to Kafka more than once.
In our test environment, Easegress generates 2GB of metrics data per day, this seems too much.
After investigation, has confirmed that `EaseMonitorMetrics` send the same metrics data to Kafka more than once.

### PR DESCRIPTION
**Pull request by localvar**
also fix several other minor issues.

### CODE REVIEW

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 217 Deploy Test Success

File: pkg/util/timetool/distributedtimer.go [hunk 1]
```diff
@@ -57,7 +57,11 @@ func (dt *DistributedTimer) run() {
 		case <-dt.done:
 			return
 		case now := <-time.After(dt.nextDurationFunc()):
-			dt.C <- now
+			// use a select to avoid block
```
Comment (xxx7xxxx) [inline pkg/util/timetool/distributedtimer.go:60]:
```suggestion
			// NOTE: Use a select-statement to avoid block and always send last time.
```

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 217 Deploy Test Success

**Review by xxx7xxxx — APPROVED** [review]

**Review by benja-wu — APPROVED** [review]
LGTM

**Pull request merged**